### PR TITLE
Release cosmos v0.5.3

### PR DESCRIPTION
**Pull request by takirala**
## High-level description

What features does this change enable? What bugs does this change fix?

[DCOS-21374](https://jira.mesosphere.com/browse/DCOS-21374) Uninstaller shouldn't assume that the "env" field exists 

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-2195](https://jira.mesosphere.com/browse/DCOS_OSS-2195) Release cosmos.

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-<number>](https://jira.mesosphere.com/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.

## Checklist for all PRs

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: This Change fixes a bug where cosmos assumed the env field was always set in marathon app definition.
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [Full Diff](https://github.com/dcos/cosmos/compare/8c04507d2f71250fa32f4565eb1467bd9e505c57...6b99be3c4e4061d94c647945034c159ba040e837)
  - [x] Test Results: [link to CI job test results for component](https://teamcity.mesosphere.io/viewLog.html?buildId=986448&buildTypeId=DcosIo_Cosmos_Scoverag&tab=testsInfo)
  - [x] Code Coverage (if available): [link to code coverage report](https://teamcity.mesosphere.io/viewLog.html?buildId=986448&buildTypeId=DcosIo_Cosmos_Scoverag&tab=buildResultsDiv)
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by takirala** [comment]
@mesosphere-mergebot bump-ee

**Comment by takirala** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by takirala** [comment]
Ping @jsancio This is ready for review

File: packages/cosmos/buildinfo.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   ],
   "single_source": {
     "kind": "url",
-    "url": "https://downloads.dcos.io/cosmos/0.5.1-18/cosmos-server-0.5.1-18-one-jar.jar",
-    "sha1": "49f855f2618deace531fb874a796568f0140bf1c"
+    "url": "https://downloads.dcos.io/cosmos/0.5.3-21/cosmos-server-0.5.3-21-one-jar.jar",
+    "sha1": "b89be2021a46664f8ebe413846e44e89dcf5564a"
```
Comment (orsenthil) [inline packages/cosmos/buildinfo.json]: Hi @takirala  - Will there be any admin-router test cases or dcos-integration tests cases applicable with this change?

When I read " When running uninstall on a service that doesn't have any environment variable configured in Marathon, it fails with the following exception."

It seems to me that we can have an integration test case with and env variable, try to install and uninstall the pacakge and assert the uninstallation does not fail.

Please let me know if this is covered else, if it is not covered, we could either cover it or add as TODO to bring in a different PR.

Comment (takirala) [reply packages/cosmos/buildinfo.json]: Thanks @orsenthil 

Created [DCOS_OSS-2221](https://jira.mesosphere.com/browse/DCOS_OSS-2221) to address this in a future pr.

We have significant number of tests covering other aspects of this functionality [here](https://github.com/dcos/cosmos/blob/master/cosmos-integration-tests/src/main/scala/com/mesosphere/cosmos/handler/UninstallHandlerSpec.scala) and I think the reason this test was missed (or not written) in first place it feels like presence `env` is minimum guarantee of the contract from outside of cosmos. Nonetheless, I have created a ticket for that. I don't know if it is straightforward to write an integration test yet, if not, we can have a unit test at the least. 
/cc @jsancio correct me if I am wrong.

Comment (orsenthil) [reply packages/cosmos/buildinfo.json]: Thanks @takirala  - that sounds good to me.

**Pull request merged**